aten/src/ATen/core/function_schema.h

```diff
@@ -36,6 +36,13 @@ struct Argument {
         kwarg_only_(kwarg_only),
         alias_info_(std::move(alias_info)),
         is_inferred_type_(is_inferred_type) {
```

Comment:
Now that the information on whether or not the type is inferred is encoded in the `type_` member, let's remove this class member (`is_inferred_type_`) and constructor argument (`is_inferred_type`) altogether.